openshift4/templates/backend.dc.yaml

```diff
@@ -335,20 +335,20 @@ objects:
                   memory: ${MEMORY_REQUEST}
               livenessProbe:
                 httpGet:
-                  path: ${BASE_PATH}/health
+                  path: ${BASE_PATH}/health/live
                   port: 5000
                   scheme: HTTP
-                initialDelaySeconds: 160
-                timeoutSeconds: 15
-                periodSeconds: 30
+                initialDelaySeconds: 80
```

Comment:
I just don't understand how a flask app can take this long to start-up, but happy to start with this and move forwards